src/types/index.tsx

```diff
@@ -1,20 +1,21 @@
-export type UserRole = "incharge" | "member" | "shift-incharge";
+export type UserRoleType = "incharge" | "member" | "shift_incharge";
```

Comment:
Why both enum and type?

Reply:
Sir , its just for readability purpose ,
we have two seprate ,
- define all types just in `type` file
- define all enums just in `constants` file 
- `UserRoleType` used only as type
- `UserRole` used only as value